build/dockerfiles/Dockerfile

```diff
@@ -17,13 +17,8 @@ RUN ./check_mandatory_fields.sh devfiles
 
 RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
-RUN chmod -R g+rwX /build/devfiles
-
-FROM docker.io/node:16.13.0-alpine3.14 as dwtemplates
-COPY ./build/scripts /build/
-COPY ./devfiles /build/devfiles
-WORKDIR /build/
 RUN ./generate_devworkspace_templates.sh
+RUN chmod -R g+rwX /build/devfiles
```

Comment:
permission problem seen in https://github.com/eclipse/che/issues/20880 could be caused by running `./generate_devworkspace_templates.sh` AFTER fixing group permissions in /build/devfiles